examples/next/faustwp-getting-started/components/NavigationMenu/NavigationMenu.js

```diff
@@ -1,9 +1,9 @@
 import classNames from 'classnames/bind';
 import { gql } from '@apollo/client';
 import Link from 'next/link';
-import flatListToHierarchical from '../../utilities/flatListToHierarchical';
 import styles from './NavigationMenu.module.scss';
 import stylesFromWP from './NavigationMenuClassesFromWP.module.scss';
+import { flatListToHierarchical } from '@faustwp/core/src';
```

Comment:
We will want users to be able to import this function directly from @faustwp/core (without the need for /src)